feature/community/src/main/res/values/strings.xml

```diff
@@ -7,4 +7,6 @@
     <string name="cardButton_services">Services</string>
     <string name="title_community">Community</string>
     <string name="default_card_description"> </string>
+    <string name="error_message_retrieving_repository_data">Error</string>
+    <string name="loading_message_retrieving_repository_data">Loading</string>
```

Comment:
I think just with the name loading_message is okay

Reply:
solved